react/admin/OrganizationsList/OrganizationsListStatusFilter/OrganizationsListStatusFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import {
+  FilterDisclosure,
+  FilterFooter,
+  FilterListbox,
+  FilterOptionCheckbox,
+  FilterPopover,
+  useFilterMultipleState,
+} from '@vtex/admin-ui'
+import { useIntl } from 'react-intl'
+
+import { organizationMessages as messages } from '../../utils/messages'
+
+const AllStatus = ['active', 'on-hold', 'inactive']
```

Comment:
what are these statuses? Is this the same as the org status? If so, shouldn't it be "pending" instead of "on-hold"?
If I understand correctly you just use it for setting the correct color around the actual status when showing the org list, but still, I believe it would be more intuitive if they have the same names.

Reply:
I'm just using the exact same names that were being used before. These are the status names that are showing up in the data that this component consumes, coming straight from graphql. If you look at the current filters in b2bstoreqa or another account with this page, you'll see that these are the values showing up there (as well as the values showing in the table for the status column). 

Ideally these should be changed to use uppercase first letter, as well as to include translation, since these values are the ones showing up on the screen regardless of the user's language. But I didn't want to change the behavior yet, just update the styles. We can do a localization pass later.